deepchem/dock/tests/test_binding_pocket.py

```diff
@@ -149,35 +149,6 @@ def test_convex_find_pockets(self):
 
     assert len(pockets) < len(all_pockets)
 
-  def test_rf_convex_find_pockets(self):
```

Comment:
So if you look down a couple of lines they were only run w/ python 2.7 which means never.  They are based off of a function which throws a ValueError("Not Implemented").  I can mark the tests as ignored?